lib/ember_cli/trailing_slash_constraint.rb

```diff
@@ -1,7 +1,7 @@
 module EmberCli
   class TrailingSlashConstraint
     def matches?(request)
-      !request.original_fullpath.to_s.ends_with?("/")
+      !request.original_fullpath.to_s.split("?")[0].ends_with?("/")
```

Comment:
Is there a test case we could add here to both demonstrate the situation and guard against regressions?